servicecontrol/servicecontrol-in-practice.md

```diff
@@ -0,0 +1,35 @@
+---
+title: ServiceControl in Practice
+summary: Details how to think about the use of ServiceControl in production environments
+tags:
+- ServiceControl
+---
+
+ServiceControl is a tool that can accommodate many situations. Features can be enabled or disabled depending on the needs of the environment. Each feature is valuable in certain contexts but also comes with a cost in terms of resource usage and performance.
+
+For example, in a development environment the [Debug Session](/servicecontrol/plugins/debug-session.md) and [Saga Audit](/servicecontrol/plugins/saga-audit.md) plugins provide detailed information to support problem analysis. Particular advise that customers not use these  plugin outside of a development environment.
+
+Making decisions about the use of the other plugins and features requires a little more thought. The temptation to 'just use them all' as a catch-all insurance policy is not a good choice. 
+
+- The cost of having certain features in play can actually cause serious performance issues without proper [capacity planning](/servicecontrol/capacity-and-planning.md). The recommended approach is that in production environments there should only be one instance of ServiceControl per machine and that it should not share resources with other potentially resource hungry processes. For example, other NServiceBus Hosts, web services or SQL Server Instances. 
+- The [Auditing](/nservicebus/operations/auditing.md) plugin will capture a record of everything that passes through an endpoint. It is worth considering turning audit off in some or all endpoints in production or directing the audit traffic to a null queue. If production audits are needed consider directing the traffic to a well resourced dedicated server. Audits in ServiceControl are primarily for performing system analysis using ServiceInsight. If ServiceInsight is not deployed in production then avoid using audits in production.
```

Comment:
Is the word "plugin"correct? Should we "feature" or similar to avoid confusing this with the SC plugins?